lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1581,6 +1592,12 @@ public function get_collection_params() {
 
 		$params['context']['default'] = 'view';
 
+		$params['after'] = array(
+			'description'        => __( 'The after parameter in a date query, send as an ISO8601 compliant date string.' ),
```

Comment:
Clients don't necessarily know what a "date query" is, given this is a WordPress-ism. Also, "after parameter" and "send" are implied in the context. What about something like:

"Limit response to resources published after a given ISO8601 compliant date."